generated/model.py

```diff
@@ -84,3 +84,10 @@ class PaginatedImageQueryList(BaseModel):
     next: Optional[AnyUrl] = Field(None, example="http://api.example.org/accounts/?page=4")
     previous: Optional[AnyUrl] = Field(None, example="http://api.example.org/accounts/?page=2")
     results: Optional[List[ImageQuery]] = None
+
+# These are reasons a label was created. A review reason is a required field when posting a human label to the API.
+class ReviewReason(str, Enum):  # noqa: N801
+    ESCALATION = "ESCALATION"
+    AUDIT = "AUDIT"
+    ASSESSMENT = "ASSESSMENT"
+    CUSTOMER_INITIATED = "CUSTOMER_INITIATED"
```

Comment:
A SDK user should only be setting `CUSTOMER_INITIATED`. The other reasons shouldn't be used for submission, so I don't think we should include them here.

(We could add them later if we return reasons in the API response, but for now, I think it would just cause confusion).

Reply:
Also, I don't think this should be in `generated/` since it's not auto-generated (and the auto-generator is gonna overwrite it next time we run it). Instead, it should be in a file under `src/groundlight/` -- I think it would be okay to put in `internalapi.py`.

Reply:
done!